src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -31,6 +31,7 @@ export const ArrayFieldTemplate: React.FC<ArrayFieldTemplateProps> = ({
   const id = idSchema.$id;
   const description = uiSchema["ui:description"] || schema.description;
   const buttonText = uiSchema["ui:buttonText"] || "Add";
+  const buttonDisabled = uiSchema["ui:buttonDisabled"] || false;
```

Comment:
This approach can be removed from this PR; I don't see it used and I added logic in #1050 so that the "Add" button is disabled if the `"ui:disabled"` option is set.

Reply:
Ah yeah it's left over from when custom jira fields was grouped with ticket creation. removed.